loadtest/control/simulcontroller/controller.go

```diff
@@ -203,6 +203,16 @@ func getActionList(c *SimulController) []userAction {
 			frequency: 0.144,
 		},
 	}
+
+	if isAdmin, err := c.user.IsSysAdmin(); err != nil && isAdmin {
```

Comment:
Unfortunately, this doesn't work here because the `roles` property of a user isn't initialized until the user logs in. So I'm afraid we have to fall back to checking this in the action itself. Also there's a mistake here, it should be `err == nil`